airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/default_paginator.py

```diff
@@ -108,14 +102,13 @@ def next_page_token(self, response: requests.Response, last_records: List[Record
         else:
             return None
 
-    def path(self):
-        if self._token and self.page_token_option and isinstance(self.page_token_option, RequestPath):
-            # Replace url base to only return the path
-            return str(self._token).replace(self.url_base.eval(self.config), "")
-        else:
-            return None
+    @abstractmethod
+    def path(self) -> Optional[str]:
+        """
+        Returns the value of the path used in the next request
+        """
 
-    def get_request_params(
+    def get_request_params(  # type: ignore[override]
```

Comment:
can you add a comment explaining why we can't get the types to match?

Reply:
I have fixed this point, no more `# type: ignore` is needed here